### ISSUE TITLE
Compatibility with Ubuntu 20.04 ?

### ISSUE THREAD

**Issue by vincegre** [storaged-project/blivet-gui#169]
Hi

Any plans to update the repository to get it compatible with the fresh new Ubuntu 20.04 ?

Thanks

Vincèn

**Comment by vojtechtrefny** [comment]
I've been neglecting the Debian/Ubuntu repository a lot lately, sorry, I'll try to do better in the future.
Anyway, I've updated the repo so it now contains newest versions of both blivet-gui and blivet and I've also enabled Ubuntu 19.10 (I didn't realize I have enable new releases manually in OBS).
There is no support for Ubuntu 20.04 yet, because it's not available in OBS, I assume it will be added after the release, but I was able to install 19.10 packages in 20.04 a everything seems to be working correctly.
